README.md

```diff
@@ -1,9 +1,10 @@
 # zowe-cli-cics-deploy-plugin
-This project aims to create a plugin for [Zowe CLI](https://github.com/zowe/zowe-cli) to deploy applications developed on a workstation to IBM CICS Transaction Server for z/OS (CICS),  Initially it will support deploying Node.js applications.  It aims to provide an experience similar to deploying to a cloud platform when deploying to CICS.  It will also provide low-level commands for performing individual steps of the deployment process that could be used as part of a CI/CD pipeline.
+This project provides a plugin for [Zowe CLI](https://github.com/zowe/zowe-cli) to deploy applications developed on a workstation to IBM CICS Transaction Server for z/OS (CICS). It aims to provide an experience similar to deploying to a cloud platform when deploying to CICS. It will also provide low-level commands for performing individual steps of the deployment process that could be used as part of a CI/CD pipeline.
```

Comment:
```suggestion
This project provides a plug-in for [Zowe CLI](https://github.com/zowe/zowe-cli) to deploy applications developed on a workstation to IBM CICS Transaction Server for z/OS (CICS). It aims to provide an experience similar to deploying to a cloud platform when deploying to CICS. It will also provide low-level commands for performing individual steps of the deployment process that could be used as part of a CI/CD pipeline.
```